### PR TITLE
Added discord "Message Embed" to site

### PR DESCRIPTION
**Pull request by humboldt123**

### CODE REVIEW

File: discord.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://discord.gg/KfpqwZB">
+  <script>location="https://discord.gg/KfpqwZB"</script>
+  <meta http-equiv="refresh" content="0; url=https://discord.gg/KfpqwZB">
+  <meta name="robots" content="noindex">
+  <h1>KAMI Blue Discord</h1>
+  <a href="https://discord.gg/KfpqwZB">Click here if you are not redirected.</a>
+    <meta content="https://kamiblue.org/favicon.png" property="og:image">
```
Comment (5HT2) [inline discord.html:10]: Can you remove this? None of the other pages use an image embed

File: discord.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <meta content="KAMI Blue" property="og:title">
+    <meta content="Join the KAMI Blue Discord: https://discord.gg/KfpqwZB" property="og:description">
+    <meta content="Join the KAMI Blue Discord: https://discord.gg/KfpqwZB" name="description">
+    <meta name="theme-color" content="#36393E"> <!-- Normally, the KAMI Blue website colour is "9b90ff". However I set it to the Discord Dark Theme background color so it looks cool and swag and hip and dope -->
+    <link href="https://discord.gg/KfpqwZB" rel="canonical">
+    <meta content="https://discord.gg/KfpqwZB" property="og:url">
+    <meta content="KAMI Blue" property="og:site_name">
+    <link href="https://kamiblue.org/favicon.png" id="favicon" rel="icon" type="image/x-icon">
```
Comment (5HT2) [inline discord.html:18]: You can replace the url to the favicon with `/favicon.png` I'm pretty sure

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: discord.html (hunk 1 above)
Comment (humboldt123) [reply discord.html:10]: I don't want to remove it. There are people/browsers with their redirects disabled.

File: discord.html (hunk 2 above)
Comment (humboldt123) [reply discord.html:18]: Yeah I'll do that.

File: discord.html (hunk 1 above)
Comment (5HT2) [reply discord.html:10]: I was talking about the image embed?

File: discord.html (hunk 2 above)
Comment (5HT2) [reply discord.html:18]: Nevermind, it doesn't work, /backdoored doesn't use it

File: discord.html (hunk 1 above)
Comment (humboldt123) [reply discord.html:10]: This isn't just for discord. There are other sites that look for that tag.

Comment (5HT2) [reply discord.html:10]: Okay, then can you add it to the default.html layout instead?

Comment (humboldt123) [reply discord.html:10]: Its already there.

**Comment by 5HT2** [comment]
Looks fine, can you revert the last commit? I was wrong, it doesn't work outside of markdown files

**Comment by humboldt123** [comment]
Done. You can merge it now.

**Pull request merged**